src/current/_includes/v23.2/storage/compaction-concurrency.md

```diff
@@ -0,0 +1 @@
+By default, the [storage engine]({% link {{ page.version.version }}/architecture/storage-layer.md %}) uses a compaction concurrency of 4, with one slot dedicated to flushes. If you have sufficient IOPs and CPU headroom, you can consider increasing this setting via the `COCKROACH_COMPACTION_CONCURRENCY` environment variable for increased performance under some workloads. We **strongly recommend** testing your workload against non-default values of this setting.
```

Comment:
The new `COCKROACH_COMPACTION_CONCURRENCY` environment variable doesn't include flushes (whereas the old undocumented rocksdb one did. So I think we should say "uses a compaction concurrency of 3", without referring to flushes.

Reply:
OK updated to 

> By default, the [storage engine]({% link {{ page.version.version }}/architecture/storage-layer.md %}) uses a compaction concurrency of 3.